elasticapm/contrib/opentracing/tracer.py

```diff
@@ -0,0 +1,96 @@
+import warnings
+
+from opentracing import Format, InvalidCarrierException, SpanContextCorruptedException, UnsupportedFormatException
+from opentracing.scope_managers import ThreadLocalScopeManager
+from opentracing.tracer import ReferenceType
+from opentracing.tracer import Tracer as TracerBase
+
+import elasticapm
+from elasticapm import instrument, traces
+from elasticapm.conf import constants
+from elasticapm.contrib.opentracing.span import OTSpan, OTSpanContext
+from elasticapm.utils import compat, disttracing
+
+
+class Tracer(TracerBase):
+    _elasticapm_client_class = elasticapm.Client
+
+    def __init__(self, client_instance=None, config=None, scope_manager=None):
+        self._agent = client_instance or self._elasticapm_client_class(config=config)
+        if scope_manager and not isinstance(scope_manager, ThreadLocalScopeManager):
+            warnings.warn(
+                "Currently, the Elastic APM opentracing bridge only supports the ThreadLocalScopeManager. "
+                "Usage of other scope managers will lead to unpredictable results."
+            )
+        self._scope_manager = scope_manager or ThreadLocalScopeManager()
+        if self._agent.config.instrument:
+            instrument()
+
+    def start_active_span(
+        self,
+        operation_name,
+        child_of=None,
+        references=None,
+        tags=None,
+        start_time=None,
```

Comment:
why bother propagating unused values (`start_time`, `ignore_active_span`)?

Reply:
Those arguments are defined by the base class in python-opentracing. And I hope to add support for them at some point.